core/server/config/index.js

```diff
@@ -179,8 +192,14 @@ ConfigManager.prototype.set = function (config) {
             availableApps:    this._config.paths.availableApps || {},
             clientAssets:     path.join(corePath, '/built/assets/')
         },
+        scheduling: {
+            postScheduling: {
```

Comment:
I'm not sure what the reasoning behind multiple kinds of nested scheduling adapters is? This seems unnecessary to me.